File: 200_code_python/25.py
# Viết bằng PYTHON
# Tìm chữ cái đầu tiên và cuối cùng của chuỗi: Nhập một chuỗi và tìm chữ cái đầu và cuối.

# Nhập chuỗi từ bàn phím
s = input("Nhập một chuỗi: ")

# Tìm chữ cái đầu và cuối của chuỗi
if len(s) > 0:
    chu_cai_dau = s[0];
    chu_cai_cuoi = s[-1] # Lấy chữ cái cuối cùng
    print(f"Chữ cái đầu tiên là: {chu_cai_dau}")
    print(f"Chữ cái cuối là: {chu_cai_cuoi}")
else:
    print("Chuỗi rỗng, không có ký tự để hiển thị.")

File: 200_code_cpp/25.cpp
// Viết bằng C++
// Tìm chữ cái đầu tiên và cuối cùng của chuỗi: Nhập một chuỗi và tìm chữ cái đầu tiên và cuối cùng.

#include <iostream>
#include <string> // Thư viện này để sử dụng kiểu dữ liệu string
using namespace std;

int main()
{
    // Nhập chuỗi từ bàn phím
    string s;
    cout << "Nhap mot chuoi: ";
    getline(cin, s); //getline() để nhập chuỗi có chưa khoảng trắng

    // Tìm chữ cái đầu và cuối
    if (!s.empty()) // Kiểm tra chuỗi có rỗng không, (emty() trả về true nếu chuỗi rỗng)
    {
        char chu_cai_dau = s[0]; // s[0] trả về ký tự đầu của chuỗi
        char chu_cai_cuoi = s[s.length() -1]; // s.length() trả về độ dài của chuỗi
        cout << "Chu cai dau tien la: " << chu_cai_dau << endl;
        cout << "Chu cai cuoi la: " << chu_cai_cuoi << endl;
    } 
    else
    {
        cout << "Chuỗi rỗng, không có ký tự để hiển thị." << endl;
    }

    return 0;
}